{"name": "Verify user can fetch single user", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'first_user_id'\n", "trace": "  File \"C:\\Users\\Jerome\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Jerome\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"BDD\\features\\steps\\tc_003_single_user.py\", line 19, in step_impl\n    endpoint = URLBuilder.format(Endpoints.USERS, user_id= context.first_user_id)\n                                                           ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Jerome\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given the API client is initialized for TC_03 First Scenario", "status": "passed", "start": 1739596070597, "stop": 1739596070597}, {"name": "When I send a Get request to fetch a single user", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'first_user_id'\n", "trace": "  File \"C:\\Users\\Jerome\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Jerome\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"BDD\\features\\steps\\tc_003_single_user.py\", line 19, in step_impl\n    endpoint = URLBuilder.format(Endpoints.USERS, user_id= context.first_user_id)\n                                                           ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Jerome\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1739596070597, "stop": 1739596070603}, {"name": "Then the response status should be 200 for TC_03 First Scenario", "status": "skipped", "start": 1739596070604, "stop": 1739596070604}, {"name": "And the response should match the updated schema for TC_03 First Scenario", "status": "skipped", "start": 1739596070604, "stop": 1739596070604}, {"name": "And the response time should be less than 2 seconds for TC_03 First Scenario", "status": "skipped", "start": 1739596070604, "stop": 1739596070604}], "start": 1739596070596, "stop": 1739596070604, "uuid": "87f460f8-770d-4403-9051-72875bce449b", "historyId": "a016b15558eff4ee295d18ed54c6965f", "fullName": "GET SINGLE USER: Verify user can fetch single user", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "regression"}, {"name": "feature", "value": "GET SINGLE USER"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}